### PR TITLE
Getting started javascript client modified to vite

### PR DESCRIPTION
**Pull request by Virgil993**

### CODE REVIEW

File: javascript/getting-started/client/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# React + Vite
```
Comment (costinsin) [inline javascript/getting-started/client/README.md:1]: Change the title to getting started

File: javascript/getting-started/client/index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/genezio.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite + React</title>
```
Comment (costinsin) [inline javascript/getting-started/client/index.html:7]: Change the title

File: javascript/getting-started/client/public/vite.svg
Comment (costinsin) [inline javascript/getting-started/client/public/vite.svg:1]: Delete this, as you don't use it anymore

File: javascript/getting-started/client/src/assets/react.svg
Comment (costinsin) [inline javascript/getting-started/client/src/assets/react.svg:1]: Delete this as you don't use it anymore

**Review by costinsin — CHANGES_REQUESTED** [review]
Please also change the readme in the workspace folder. It now states that the client can be started with 'npm start' which is not true. Replace with 'npm run dev'

File: javascript/getting-started/client/package.json [hunk 3]
```diff
@@ -1,38 +1,37 @@
...
+    "build": "vite build",
+    "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview",
+    "install-prod-sdk": "npm install @genezio-sdk/getting-started-genezio_us-east-1@1.0.0-prod"
+  },
   "dependencies": {
-    "@testing-library/jest-dom": "^5.16.5",
-    "@testing-library/react": "^13.4.0",
-    "@testing-library/user-event": "^13.5.0",
+    "@genezio-sdk/getting-started-genezio_us-east-1": "^1.0.0-prod",
```
Comment (vladiulianbogdan) [inline javascript/getting-started/client/package.json:14]: this dependency should be removed otherwise if the user runs `genezio local` it will fail

**Review by stefandarius — APPROVED** [review]

**Pull request merged**